dev-resources/private/test/src/cljc/foo/bar/core.cljc

```diff
@@ -0,0 +1,16 @@
+(ns foo.bar.core
+  (#?(:clj  :require
+      :cljs :require-macros)
+    [foo.bar.macros]))
```

Comment:
Why is this in a new line, is it idiomatic like this? To me it should go on the same line as `:require`..